app/views/agenda/agenda.html.erb

```diff
@@ -1,26 +1,29 @@
-
-<div class="container mt-5 text-center">
+<div class="simple-calendar">
+ <div class="container mt-5 text-center">
```

Comment:
Please fix your indentation here, remember we use 2 spaces not 1. Maybe you need to fix all lines insiede of your new div.